### PR TITLE
fixed typo plt/racket to racket/racket

### PR DESCRIPTION
**Pull request by avelino**
repo migrate on release 6.2
https://github.com/racket/racket/releases/tag/v6.2

### CODE REVIEW

File: announcements/v6.2.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ and many other package repos, all found on GitHub. If you have time and
 if you wish to get involved, please take a look at the GitHub repos and
 find your favorite places to learn, fix, and enhance our world.
 
-The core repo is at https://github.com/plt/racket, and the
+The core repo is at https://github.com/racket/racket, and the
```
Comment (stamourv) [inline announcements/v6.2.txt]: I don't think this one should change.
The 6.2 announcement is at this point more of a "historical document", and so having it reflect things as they were then is the right thing, IMO.

File: blog/_src/posts/2012-11-25-tutorial-contributing-to-racket.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
     Title:Tutorial: Contributing to Racket
     Date:2012-11-25T17:02:00.001-05:00
-    Tags: 
+    Tags:
```
Comment (stamourv) [inline blog/_src/posts/2012-11-25-tutorial-contributing-to-racket.md]: Could you take out the whitespace changes from your commit?

File: blog/_src/posts/2015-06-19-racket-v62.md [hunk 3]
```diff
@@ -13,11 +13,11 @@ With this release we are taking a major step forward to get our user community e
 
 
 
-The core repo is at [https://github.com/plt/racket](https://github.com/plt/racket), and the package repos are listed at [https://github.com/racket/](https://github.com/racket/).
+The core repo is at [https://github.com/racket/racket](https://github.com/racket/racket), and the package repos are listed at [https://github.com/racket/](https://github.com/racket/).
```
Comment (stamourv) [inline blog/_src/posts/2015-06-19-racket-v62.md]: This one is probably good to change, though. It's a link, and it's on the blog.

File: announcements/v6.2.txt (hunk 1 above)
Comment (avelino) [reply announcements/v6.2.txt]: @stamourv fixed you comments

**Comment by stamourv** [comment]
Merged, thanks!

**Pull request closed without merging**